.hooks/pre-push

```diff
@@ -14,7 +17,18 @@ npx eslint --quiet
 printf "\n⏳ npm run test\n"
 npm run test
 
-printf "\n⏳ composer test\n"
-vendor/bin/phpunit
+# Check the OS. Windows does not support the --parallel flag.
+if [ "$(uname)" = "Linux" ] || [ "$(uname)" = "Darwin" ]; then
+  # Use --parallel if macOS or Linux are detected.
+  printf "\n⏳ composer test -- --parallel\n"
+  vendor/bin/paratest &
```

Comment:
Why run this in the background (`&`)? It doesn't seem necessary, and you could avoid the trap/wait.
```
jamiras@ubuntu24 pr/2652:~/source/RAWeb$ vendor/bin/paratest
ParaTest v7.4.0 upon PHPUnit 10.5.10 by Sebastian Bergmann and contributors.

Processes:     4
Runtime:       PHP 8.2.22
Configuration: /home/jamiras/source/RAWeb/phpunit.xml

...............................................................  63 / 385 ( 16%)
............................................................... 126 / 385 ( 32%)
............................................................... 189 / 385 ( 49%)
............................................................... 252 / 385 ( 65%)
............................................................... 315 / 385 ( 81%)
............................................................... 378 / 385 ( 98%)
.......                                                         385 / 385 (100%)

Time: 00:27.435, Memory: 26.00 MB

OK (385 tests, 2470 assertions)
jamiras@ubuntu24 pr/2652:~/source/RAWeb$ vendor/bin/phpunit
PHPUnit 10.5.10 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.2.22
Configuration: /home/jamiras/source/RAWeb/phpunit.xml

...............................................................  63 / 385 ( 16%)
............................................................... 126 / 385 ( 32%)
............................................................... 189 / 385 ( 49%)
............................................................... 252 / 385 ( 65%)
............................................................... 315 / 385 ( 81%)
............................................................... 378 / 385 ( 98%)
.......                                                         385 / 385 (100%)

Time: 01:33.493, Memory: 123.00 MB

OK (385 tests, 2470 assertions)
```

Reply:
When I was making this update, I encountered a bizarre issue where paratest ran in the background and could not be terminated, even after killing the foreground pre-push hook process.

I don't seem to be having this problem anymore though 🤔

I've removed the `trap`, `wait`, and `&` stuff in latest.